configuration/reports/reportdescriptors/dataexports/sql/checkin.sql

```diff
@@ -0,0 +1,77 @@
+SELECT encounter_type_id INTO @enctype FROM encounter_type et WHERE uuid='55a0d3ea-a4d7-4e88-8f01-5aceb2d3c61b';
+
+DROP TABLE IF EXISTS checkin_details;
```

Comment:
make this a temporary table so it gets cleaned up

Reply:
I see its already temporary 
<img width="509" alt="image" src="https://github.com/PIH/openmrs-config-pihliberia/assets/24842483/fa05f61b-1e09-45c7-8b75-84dc7e8ce3d2">